src/test/java/net/minecraftforge/debug/entity/living/LivingFoodEventTest.java

```diff
@@ -0,0 +1,58 @@
+package net.minecraftforge.debug.entity.living;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.mojang.datafixers.util.Pair;
+
+import net.minecraft.world.effect.MobEffectInstance;
+import net.minecraft.world.effect.MobEffects;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.living.LivingEatEvent;
+import net.minecraftforge.event.entity.living.LivingFoodEffectEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("living_food_event_test")
+public class LivingFoodEventTest
+{
+    public static final Logger LOGGER = LogManager.getLogger();
+    public LivingFoodEventTest()
+    {
+        MinecraftForge.EVENT_BUS.addListener(this::onLivingEat);
+        MinecraftForge.EVENT_BUS.addListener(this::onLivingFoodEffect);
+    }
+
+    public void onLivingEat(LivingEatEvent event)
+    {
+    	LOGGER.info("{} is about to eat {} with {} nutrition and {} saturation", 
```

Comment:
Mixes of tabs and spaces throughout doc, fix.

Reply:
What is the standard for using tabs or spaces in all files?

Reply:
Tabs are not permitted